build-support/bin/release.sh

```diff
@@ -585,21 +581,19 @@ function build_pex() {
           ;;
       esac
       local platforms=("${platform}")
-      local dest="${ROOT}/dist/pants.${PANTS_UNSTABLE_VERSION}.${platform}.${dest_suffix}"
-      local stable_dest="${DEPLOY_DIR}/pex/pants.${PANTS_STABLE_VERSION}.${platform}.${dest_suffix}"
+      local dest="${ROOT}/dist/pants.${PANTS_UNSTABLE_VERSION}.${platform}.pex"
```

Comment:
So is the pex being built with both 3.6 and 3.7 non-abi3 platform specific wheels included?:
```
$ unzip -qc ~/Desktop/pants.1.21.0.dev0.py36.pex PEX-INFO | jq '.distributions | keys | map(select(. | test("cp36")))'
[
  "PyYAML-5.1.2-cp36-cp36m-linux_x86_64.whl",
  "PyYAML-5.1.2-cp36-cp36m-macosx_10_11_x86_64.whl",
  "cffi-1.11.5-cp36-cp36m-macosx_10_6_intel.whl",
  "cffi-1.11.5-cp36-cp36m-manylinux1_x86_64.whl",
  "pantsbuild.pants-1.21.0.dev0+git87843f72-cp36-abi3-linux_x86_64.whl",
  "pantsbuild.pants-1.21.0.dev0+git87843f72-cp36-abi3-macosx_10_11_x86_64.whl",
  "psutil-5.6.3-cp36-cp36m-linux_x86_64.whl",
  "psutil-5.6.3-cp36-cp36m-macosx_10_11_x86_64.whl",
  "python_Levenshtein-0.12.0-cp36-cp36m-linux_x86_64.whl",
  "python_Levenshtein-0.12.0-cp36-cp36m-macosx_10_11_x86_64.whl",
  "pywatchman-1.4.1-cp36-cp36m-linux_x86_64.whl",
  "pywatchman-1.4.1-cp36-cp36m-macosx_10_11_x86_64.whl",
  "setproctitle-1.1.10-cp36-cp36m-linux_x86_64.whl",
  "setproctitle-1.1.10-cp36-cp36m-macosx_10_11_x86_64.whl",
  "thriftpy2-0.4.5-cp36-cp36m-linux_x86_64.whl",
  "thriftpy2-0.4.5-cp36-cp36m-macosx_10_11_x86_64.whl"
]
```
If not we need to keep on generating a py36.pex and add a py37.pex.

Reply:
The goal is for it to include Py36 _and_ Py37 non-abi3 wheels so that we can release one single fat PEX.

Reply:
Yes, but is it? I don't think so without specifying multiple explicit platforms. In this case we'll need 4: 3.6 osx, 3.7 osx, 3.6 linux, 3.7 linux. From my phone so I don't have the proper platform strings at hand, but you get the gist I think.